ui/src/textbox.tsx

```diff
@@ -96,7 +111,10 @@ export const
         suffix: m.suffix,
         multiline: m.multiline,
         spellCheck: m.spellcheck,
-        type: m.password ? 'password' : undefined,
+        type: m.password ? 'password' : (m.keyboard || 'text'),
+        styles,
+        onWheel: m.keyboard === 'number' ? e => e.currentTarget.blur() : undefined, // Disable setting number with scroll.
+        onKeyDown: m.keyboard === 'number' ? e => e.preventDefault() : undefined, // Disable setting number with keyboard arrows.
```

Comment:
Keep the native browser behavior - https://github.com/h2oai/wave/issues/2170#issuecomment-1781226049

Reply:
> then hack around the undesired buttons/scroll via a little bit of custom CSS and JS

Based on the claim from the comment mentioned above I understand arrows and scrolling as undesired or am I missing something?

Reply:
Let's add the needed attribute and then hack around the undesired buttons/scroll via a little bit of **custom CSS and JS that I can send you**. I would prefer to **not bake this into Wave for now**.

Reply:
Oh, I see. So by "custom css/js" you mean css/js used within a python app. And by "not bake this into Wave" you are talking about **custom css/js** and not about **arrows and scrolling**.